georocket-cli/src/main/java/io/georocket/GeoRocketCli.java

```diff
@@ -159,6 +147,25 @@ public static void main(String[] args) throws IOException {
     }
   }
 
+  /**
+   * Setup GeoRocket Cli.
+   */
+  protected static void setup() {
```

Comment:
static method cannot be overridden in Java, so why this change? Wouldn't it make more sense to reduce the `main` method to only two lines?

```
GeoRocketCli cli = new GeoRocketCli();
cli.run(args);
```

Or something similar?